packages/venia-ui/lib/components/MagentoRoute/magentoRoute.js

```diff
@@ -13,7 +13,12 @@ const MESSAGES = new Map()
     .set(INTERNAL_ERROR, 'Something went wrong. Please try again.');
 
 const MagentoRoute = () => {
-    const talonProps = useMagentoRoute();
+    const magentoRouteProps = {};
+    // If we have a specific store view code configured pass it into the url resolver
+    if (process.env.STORE_VIEW_CODE) {
+        magentoRouteProps.store = process.env.STORE_VIEW_CODE;
+    }
```

Comment:
Probably we can use recently added global `STORE_VIEW_CODE` - https://github.com/magento/pwa-studio/pull/2643/files#diff-1acbde853507ec5d5dac59ef5301a593R62

Reply:
Thanks for the heads up!